deploy/example_catalog/cr-app-tensorflowcpu.json

```diff
@@ -53,11 +53,11 @@
             }
         ],
         "defaultConfigPackage": {
-            "packageURL" : "http://16.143.21.166/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
+            "packageURL" : "http://16.143.21.22/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
```

Comment:
This should be removed, rebuild the image with app config as part of the image. appconfig can live under /opt/configscript, then replace this path with "/opt/configscript/appconfig-1.8.tgz

Reply:
Noted. Will do.